kconfig

```diff
@@ -1578,12 +1492,7 @@ CONFIG_ARCH_USE_SYM_ANNOTATIONS=y
 # CONFIG_CRC16 is not set
 # CONFIG_CRC_T10DIF is not set
 # CONFIG_CRC_ITU_T is not set
-CONFIG_CRC32=y
```

Comment:
Did you do this manually? From memory I think this gets selected from other subsystems no?